tests/testthat/test-theme.r

```diff
@@ -202,6 +202,19 @@ test_that("All elements in complete themes have inherit.blank=TRUE", {
   expect_true(inherit_blanks(theme_void()))
 })
 
+test_that("Elements can be merged", {
+  text_base <- element_text(colour = "red", size = 10)
+  expect_equal(merge_element(element_text(colour = "blue"), text_base),
```

Comment:
Can you please use tidyverse style indenting, e.g. http://style.tidyverse.org/syntax.html#long-lines